crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -90,6 +90,17 @@ def addons
       end
     end
 
+    def repocheck
+      {
+        cloud: {
+          available: repo_version_available("suse-openstack-cloud", "7") == 0
```

Comment:
right, actually the backport is important here. but what would be the sles repo version for master? Do we know yet?